index.js

```diff
@@ -0,0 +1,35 @@
+// npm module entry
+//
+// Provides OpenMoji data in a web developer friendly fashion.
+//
+const version = require('./package.json').version
+const openmojisRaw = require('./data/openmoji.json')
+const colorPalette = require('./data/color-palette.json')
+// For path joining
+const path = require('path')
+// Path to SVG directories
+// TODO expose svg files under src/:group/:subgroups?
```

Comment:
?

Reply:
There seems to be a duplicate set of coloured SVG emoji files released within the npm package. I assume the ones under `src/` go through some kind of optimisation before ending up in `color/svg/`. The ones under `src/` could be left out from the npm package by appending the dir to `.npmignore` file. That would reduce npm package size that already is quite big.

Reply:
Yes. Aware of it. We decided to ship with the src files ... but happy to discuss this. I don't think anyone is using the src files via npm install. I'm happy to do either way. It would save us ca. 7 mb of file size footprint.

@carlinmack What do you think? Should we exclude the src folder from the npm package y/n?

Reply:
I'm not sure, wasn't there another discussion about whether people should generate assets or not? 

I think, if we can, we should bundle the color/black svgs so that developers can start using the package out of the box rather than having to generate assets which may cause problems. We should also have a note somewhere that 72x72 will improve performance but should only be used at sizes under X.

Reply:
What about moving it to a new issue, so it won't block this PR, then?